hieradata/cluster/pukem.yaml

```diff
@@ -0,0 +1,6 @@
+---
+clustershell::groupmembers:
+  pukem: {group: "pukem", member: "pukem[01-04]"}
+profile::core::ospl::enable_rundir: true
```

Comment:
this is for the OSPL daemon not really needed here

Reply:
good catch, removed.